src/main/java/com/udemy/elearning/services/CheckoutCourseService.java

```diff
@@ -0,0 +1,42 @@
+package com.udemy.elearning.services;
+
+
+import com.udemy.elearning.dto.CheckoutCourseRequest;
+import com.udemy.elearning.models.CheckoutCourse;
+import com.udemy.elearning.repository.CheckoutCourseRepository;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+import org.webjars.NotFoundException;
+
+import java.util.List;
+
+@Service
+public class CheckoutCourseService {
+
+    private static final Logger logger = LogManager.getLogger(CheckoutCourseService.class);
+
+    private final CheckoutCourseRepository checkoutCourseRepository;
+
+    public CheckoutCourseService(CheckoutCourseRepository checkoutCourseRepository) {
+        this.checkoutCourseRepository = checkoutCourseRepository;
+    }
+
+    public CheckoutCourse create(CheckoutCourseRequest checkoutCourseRequest){
+        CheckoutCourse checkoutCourse = new CheckoutCourse();
+        checkoutCourse.setCourseId(checkoutCourse.getCourseId());
+        checkoutCourse.setPrice(checkoutCourse.getPrice());
+        CheckoutCourse checkoutCourseResult = checkoutCourseRepository.save(checkoutCourse);
```

Comment:
you have save twice  CheckoutCourse checkoutCourseResult = checkoutCourseRepository.save(checkoutCourse);
        return  checkoutCourseRepository.save(checkoutCourse);

Reply:
Yes bong, this mean we can we can checkout with multiple course.